db/migrate/20150213103214_add_additional_location_fields.rb

```diff
@@ -0,0 +1,7 @@
+class AddAdditionalLocationFields < ActiveRecord::Migration
+  def change
+    rename_column 'people', 'location', 'location_in_building'
+    add_column 'people', 'building', :text
+    add_column 'people', 'city', :text
```

Comment:
I just noticed that we're mixing strings and symbols in our migrations. But it's no biggie.